packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1190,8 +1190,8 @@ export const UnknownCLIError = {
 export const GenerateContentTypesError = {
 	name: 'GenerateContentTypesError',
 	title: 'Failed to generate content types.',
-	message: (errorMessage: string) =>
-		`\`astro sync\` command failed to generate content collection types: ${errorMessage}`,
+	message: (errorMessage: string, file?: string) =>
```

Comment:
You're gonna need a `@message` property here, because the docs won't be able to parse a message from the ternary.